server/application-server/src/main/java/de/tum/in/www1/hephaestus/leaderboard/LeaderboardService.java

```diff
@@ -88,6 +92,8 @@ public List<LeaderboardEntry> createLeaderboard(Optional<LocalDate> after, Optio
             return new LeaderboardEntry(user.getLogin(), user.getAvatarUrl(), user.getName(), user.getType(),
                     score.get(),
                     0, // preliminary rank
+                    reviewedPRs.size(),
+                    numberOfReviews.get(),
```

Comment:
```suggestion
                    user.getReviews().size(),
```
I think you can get rid of the counter variable

Reply:
Absolutely! Done in d9f14d6.